src/pages/RoomMembersPage.tsx

```diff
@@ -121,45 +88,41 @@ function RoomMembersPage(props) {
      *
      * @param {String} login
      */
-    const removeUser = useCallback((accountID) => {
-        setSelectedMembers((prevSelected) => _.without(prevSelected, accountID));
+    const removeUser = useCallback((accountID: number) => {
+        setSelectedMembers((prevSelected) => prevSelected.filter((selected) => selected !== accountID));
     }, []);
 
-    /**
-     * Toggle user from the selectedMembers list
-     *
-     * @param {String} accountID
-     * @param {String} pendingAction
-     *
-     */
+    /** Toggle user from the selectedMembers list */
     const toggleUser = useCallback(
-        (accountID, pendingAction) => {
-            if (pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
+        ({accountID, pendingAction}: User) => {
+            if (pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE || !accountID) {
                 return;
             }
 
             // Add or remove the user if the checkbox is enabled
-            if (_.contains(selectedMembers, Number(accountID))) {
-                removeUser(Number(accountID));
+            if (selectedMembers.includes(accountID)) {
+                removeUser(accountID);
             } else {
-                addUser(Number(accountID));
+                addUser(accountID);
             }
         },
         [selectedMembers, addUser, removeUser],
     );
 
-    /**
-     * Add or remove all users passed from the selectedMembers list
-     * @param {Object} memberList
-     */
-    const toggleAllUsers = (memberList) => {
-        const enabledAccounts = _.filter(memberList, (member) => !member.isDisabled);
-        const everyoneSelected = _.every(enabledAccounts, (member) => _.contains(selectedMembers, Number(member.keyForList)));
+    /** Add or remove all users passed from the selectedMembers list */
+    const toggleAllUsers = (memberList: User[]) => {
+        const enabledAccounts = memberList.filter((member) => !member.isDisabled);
+        const everyoneSelected = enabledAccounts.every((member) => {
+            if (!member.accountID) {
+                return false;
+            }
+            return selectedMembers.includes(member.accountID);
```

Comment:
Since you updated the logic (use `accountID` instead of `keyForList`), make sure the functionality works as expected.

Reply:
yep, keyForList is actually string created by String(accountId)
` result.push({
                keyForList: String(accountID),`
so instead of doing Number(keyForList) I'm doing accountId
Made sure everything works as expected 😃